tests/integration/test_storage_s3_queue/test.py

```diff
@@ -263,15 +302,31 @@ def test_delete_after_processing(started_cluster, mode):
         ).splitlines()
     ] == sorted(total_values, key=lambda x: (x[0], x[1], x[2]))
 
-    minio = started_cluster.minio_client
-    objects = list(minio.list_objects(started_cluster.minio_bucket, recursive=True))
-    assert len(objects) == 0
+    if engine_name == "S3Queue":
+        minio = started_cluster.minio_client
+        objects = list(minio.list_objects(started_cluster.minio_bucket, recursive=True))
+        assert len(objects) == 0
+    else:
+        client = started_cluster.blob_service_client.get_container_client(
+            AZURE_CONTAINER_NAME
+        )
+        objects_iterator = client.list_blobs(files_path)
+        for objects in objects_iterator:
+            assert False
 
 
-@pytest.mark.parametrize("mode", AVAILABLE_MODES)
-def test_failed_retry(started_cluster, mode):
+@pytest.mark.parametrize(
```

Comment:
you can write `@pytest.mark.parametrize` multiple times
each combination will be run
```py
@pytest.mark.parametrize(argnames: "mode", argvalues: ["unordered", "ordered"])
@pytest.mark.parametrize(argnames: "engine_name", argvalues: ["S3Queue", "AzureQueue"])
```

Reply:
It does not seem to work for me:
```
E     File "/home/kssenii/workspace/ClickHouse1/tests/integration/test_storage_s3_queue/test.py", line 252
E       @pytest.mark.parametrize(argnames: "mode", argvalues: ["unordered", "ordered"])
E                                        ^
E   SyntaxError: invalid syntax
```

Reply:
I'm an idiot and copied the things my editor shows
```py
@pytest.mark.parametrize("mode", ["unordered", "ordered"])
@pytest.mark.parametrize("engine_name", ["S3Queue", "AzureQueue"])
```